### ISSUE TITLE
accessibility: elements of the interface lack necessary labels

### ISSUE THREAD

**Issue by Piciok** [Tlaster/Tuuto#4]
Currently the controls of the app aren't labelled for screenreaders used by blind people. This way it is impossible to read toots or distinguish between the actions performed by the accompanying buttons.
